content/news/this-week-in-redox-30.md

```diff
@@ -0,0 +1,198 @@
++++
+title = "This Week in Redox 30"
+author = "goyox86"
+date = "2017-10-13 11:57:31 +0100"
++++
+
+This is the 30th post of a series of blog posts tracking the development and progress of Redox, the Rust operating system. If you want to know more about Redox in general, visit our [Github page](https://github.com/redox-os/redox).
+
+*(edited by [@goyox86](https://github.com/goyox86))*
+
+# PSA
+
+If you have any questions, ideas, or are curious about Redox, we recommend joining `#redox` on `irc.mozilla.org`, [our Discourse forum](https://discourse.redox-os.org/) or you can get an invite to our chat by sending an email request to `info@redox-os.org`.
+
+# What's new in Redox?
+
+## TL;DR
+
+Hello there! And welcome to another edition of TWiR! 
+
+As usual, we have been busy making Redox better and we have a couple of exciting news to share. So, Let's get started!
+
+First of all, [@andre](https://github.com/andre-richter) pointed out that the Docker experience has been improved. Among others, consecutive builds are now way faster because `cargo` downloads are cached with named volumes. Also, there's documentation for running the container interactively now. Please check it out [here](https://github.com/redox-os/redox/blob/master/docker/README.md)!
+
+We are super excited about the work done by [@jackpot51](https://github.com/jackpot51) towards the installer support. For example, the **bootloader** now supports filesystem UUID detection and is able to send that information to the kernel at boot time. The **kernel** has notably gained the ability to receive environment variables at the main entry point (`kmain`) this is needed to being able to receive that `REDOXFS_UUID` environment with the filesystem UUID to be mounted. It's also worth mentioning that [@jackpot51](https://github.com/jackpot51) made some refactoring to the **kernel** arguments infrastructure. 
+
+The other very important change is related to schemes. Redox now supports hierarchical schemes enabling for example, more than one disk controller.
+
+Other exciting news in the **kernel** is the work on capability based security. Basically, capability mode will make a process unable to open new file descriptors unless it has required access to it, improving security and allowing more reasoning about the system. The `sys:iostat` scheme for example, should tell you what any process is allowed to do, once it is in capability mode. You can read more about capability based security [here](https://en.wikipedia.org/wiki/Capability-based_security) and also check out [FreeBSD's implementation of this model](https://wiki.freebsd.org/Capsicum).
+
+Along with this work is the support for disk hierarchies added to the AHCID **driver** and the support for capability mode added to all the **drivers** by using the `null` scheme. In addition to that, we have now a `vmmouse` driver which should improve significantly the mouse support in `QEMU`.
```

Comment:
It isn't the `null` scheme, it is the `0` namespace - which can also be called the "null namespace", but not the "`null` namespace"